src/data/mv_polynomial.lean

```diff
@@ -925,6 +925,47 @@ lemma map_sub : (p - q).map f = p.map f - q.map f := is_ring_hom.map_sub _
 
 end map
 
+section aeval
+
+/-- The algebra of multivariate polynomials. -/
+instance mv_polynomial (R : Type u) [comm_ring R]
+  (σ : Type v) : algebra R (mv_polynomial σ R) :=
+{ to_fun := mv_polynomial.C,
+  commutes' := λ _ _, mul_comm _ _,
+  smul_def' := λ c p, (mv_polynomial.C_mul' c p).symm,
+  .. mv_polynomial.module }
+
+variables (R : Type u) (A : Type v) (f : σ → A)
+variables [comm_ring R] [comm_ring A] [algebra R A]
+
+/-- (ι → A) → Hom[R-Alg](R[ι],A) -/
```

Comment:
I just noticed that both this and the docstring of `aeval` in `data.polynomial` end up linkified in markdown ([see here](https://leanprover-community.github.io/mathlib_docs/ring_theory/algebra.html#mv_polynomial.aeval)). 

Some possible solutions:
- add a space or other character between the square brackets and parens
- use  different characters in place of square brackets
- use LaTeX and make `R-Alg` a subscript

Reply:
It would be nice if someone with knowledge in category theory will rewrite this docstring so that it will be both correct from the category theory point of view and helpful to those who don't care about categories. For me, something like "a map `σ → A` from a set to an algebra over `R` generates an `R`-algebra homomorphism from multivariate polynomials over `σ` to `A`" would be more helpful.

Reply:
I wholeheartedly agree: giving an explanation in English would be even better than just fixing the formatting.

Reply:
This sentence should go together with some short sentence that aimed at those familiar with category theory, and I can't to this part.

Reply:
This one here is particularly confusing since the `(ι → A)` in the docstring is the `(f : σ → A)` in the variables block above.  Not only are the names different, the type in the declaration and the docstring don't match up when you read the source code.

Reply:
Ouch, I've updated definition but not docstring.